finetuner/experiment.py

```diff
@@ -180,6 +181,79 @@ def create_run(
         )
         return run
 
+    def _prepare_data(
+        self,
+        train_data: Union[DocumentArray, str],
+        eval_data: Union[DocumentArray, str],
+        eval_callback: EvaluationCallback,
+        run_name: str,
+    ):
+        """Upload data to Hubble and returns their names.
+
+        Uploads all data needed for fine-tuning - training data,
+        evaluation data and query/index data for `EvaluationCallback`.
+
+        Checks not to upload same dataset twice.
+
+        :param train_data: Either a `DocumentArray` for training data or a
+            name of the `DocumentArray` that is pushed on Hubble.
+        :param train_data: Either a `DocumentArray` for evaluation data or a
+            name of the `DocumentArray` that is pushed on Hubble.
+        :param eval_callback: Evaluation callback that contains query and index data
+            and gets modified in-place.
+        :param run_name: Name of the run.
+        :return: Name(s) of the uploaded data.
```

Comment:
I am not sure, if it is good to move the documentation of the arguments into this private function. Maybe it is better to move it into the public `create_run` function, where it is more visible and it is also the only point where this function is used.

Reply:
not sure what you mean.. `create_run` still has the documentation of its arguments.